code/modules/jobs/job_types/roguetown/nobility/lord.dm

```diff
@@ -28,7 +28,7 @@ GLOBAL_LIST_EMPTY(lord_titles)
 	whitelist_req = FALSE
 	min_pq = 10
 	max_pq = null
-	give_bank_account = 1000
+	give_bank_account = TRUE
```

Comment:
I have to get back to this:
![image](https://github.com/user-attachments/assets/6af6a738-41dd-4041-ba7f-0bcfd5bfff0b)

What this equates to is supposedly initial deposit.

If the value is FALSE they get no account, if the value is TRUE they get an empty account, if the value is superior to 1 they get an initial deposit.

What I'm asking you is that you standardize this var on other jobs:

![image](https://github.com/user-attachments/assets/8dca61da-e117-408c-a43d-e9db64080932)

As they seem to have random numbers.

Even I fucked it up and gave the seamstress 2 for absolutely no reason.

![image](https://github.com/user-attachments/assets/f451d513-5516-4496-ace3-2f8485f55367)

What I think happened here is, some people who touched this var, like me didn't understand its purpose fully and just thought it meant "bank account number".

I also ask that you add a commend above the var to describe what it does in _job.dm like:

![image](https://github.com/user-attachments/assets/2105c7c7-80dc-413c-ba90-8e947330fd7b)

Cheers!

Reply:
To pile on this, there, in my view, isnt much sense in givign working classes an initial investment since they already spawn with mammons on their person.